sdk/samples/iot/hub/src/generate_certificate.cmd

```diff
@@ -6,45 +6,53 @@
 openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
 IF %ERRORLEVEL% NEQ 0 ( 
 	echo Failed generating certificate key
-	exit b 1
+	exit b 1@REM Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
error here?

Reply:
yea delete everything before the `@REM` on this line and it looks like it works on my box

Reply:
oops. Done.